osu.Game/Graphics/UserInterface/SliderSounds.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.Graphics;
+using osu.Framework.Utils;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public partial class SliderSounds<T> : Component
```

Comment:
I'm not sure the generic part is required here. Can you just check the value equality in the call site instead?

Reply:
I'd also rename this to `SliderAdjustTick` or something like that.

Reply:
> I'm not sure the generic part is required here. Can you just check the value equality in the call site instead?

done, but i was somewhat hesistant in doing so, i feel it was better than the alternative, not that it really matters.